src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -271,15 +271,31 @@ const WorkspaceDashboard = _.flow(
 
   const loadStorageCost = withErrorReporting('Error loading storage cost data', async () => {
     if (Utils.canWrite(accessLevel)) {
-      const { estimate, lastUpdated } = await Ajax(signal).Workspaces.workspace(namespace, name).storageCostEstimate()
-      setStorageCost({ estimate, lastUpdated })
+      try {
+        const { estimate, lastUpdated } = await Ajax(signal).Workspaces.workspace(namespace, name).storageCostEstimate()
+        setStorageCost({ estimate, lastUpdated })
```

Comment:
```suggestion
        setStorageCost({ isSuccess: true, estimate, lastUpdated })
```

Needed to make the "Update on <date>" message appear.

Reply:
Thanks!